// sib-api-v3-sdk/src/index.js
/*
 * SendinBlue API
 * SendinBlue provide a RESTFul API that can be used with any languages. With this API, you will be able to :   - Manage your campaigns and get the statistics   - Manage your contacts   - Send transactional Emails and SMS   - and much more...  You can download our wrappers at https://github.com/orgs/sendinblue  **Possible responses**   | Code | Message |   | :-------------: | ------------- |   | 200  | OK. Successful Request  |   | 201  | OK. Successful Creation |   | 202  | OK. Request accepted |   | 204  | OK. Successful Update/Deletion  |   | 400  | Error. Bad Request  |   | 401  | Error. Authentication Needed  |   | 402  | Error. Not enough credit, plan upgrade needed  |   | 403  | Error. Permission denied  |   | 404  | Error. Object does not exist |   | 405  | Error. Method not allowed  |   | 406  | Error. Not Acceptable  | 
 *
 * OpenAPI spec version: 3.0.0
 * Contact: contact@sendinblue.com
 *
 * NOTE: This class is auto generated by the swagger code generator program.
 * https://github.com/swagger-api/swagger-codegen.git
 *
 * Swagger Codegen version: 2.4.26
 *
 * Do not edit the class manually.
 *
 */

(function(factory) {
  if (typeof define === 'function' && define.amd) {
    // AMD. Register as an anonymous module.
    define(['ApiClient', 'model/AbTestCampaignResult', 'model/AbTestCampaignResultClickedLinks', 'model/AbTestCampaignResultStatistics', 'model/AbTestVersionClicks', 'model/AbTestVersionClicksInner', 'model/AbTestVersionStats', 'model/AddChildDomain', 'model/AddContactToList', 'model/AddCredits', 'model/BlockDomain', 'model/Body', 'model/Body1', 'model/Body10', 'model/Body11', 'model/Body12', 'model/Body2', 'model/Body3', 'model/Body4', 'model/Body5', 'model/Body6', 'model/Body7', 'model/Body8', 'model/Body9', 'model/CompaniesList', 'model/Company', 'model/CompanyAttributes', 'model/CompanyAttributesInner', 'model/ConversationsMessage', 'model/ConversationsMessageFile', 'model/ConversationsMessageFileImageInfo', 'model/CreateAttribute', 'model/CreateAttributeEnumeration', 'model/CreateChild', 'model/CreateContact', 'model/CreateDoiContact', 'model/CreateEmailCampaign', 'model/CreateEmailCampaignRecipients', 'model/CreateEmailCampaignSender', 'model/CreateList', 'model/CreateModel', 'model/CreateReseller', 'model/CreateSender', 'model/CreateSenderIps', 'model/CreateSenderModel', 'model/CreateSmsCampaign', 'model/CreateSmsCampaignRecipients', 'model/CreateSmtpEmail', 'model/CreateSmtpTemplate', 'model/CreateSmtpTemplateSender', 'model/CreateSubAccount', 'model/CreateSubAccountResponse', 'model/CreateUpdateContactModel', 'model/CreateUpdateFolder', 'model/CreateWebhook', 'model/CreatedProcessId', 'model/Deal', 'model/DealAttributes', 'model/DealAttributesInner', 'model/DealsList', 'model/DeleteHardbounces', 'model/EmailExportRecipients', 'model/ErrorModel', 'model/FileData', 'model/FileDownloadableLink', 'model/FileList', 'model/GetAccount', 'model/GetAccountMarketingAutomation', 'model/GetAccountPlan', 'model/GetAccountRelay', 'model/GetAccountRelayData', 'model/GetAggregatedReport', 'model/GetAttributes', 'model/GetAttributesAttributes', 'model/GetAttributesEnumeration', 'model/GetBlockedDomains', 'model/GetCampaignOverview', 'model/GetCampaignRecipients', 'model/GetCampaignStats', 'model/GetChildAccountCreationStatus', 'model/GetChildDomain', 'model/GetChildDomains', 'model/GetChildInfo', 'model/GetChildInfoApiKeys', 'model/GetChildInfoApiKeysV2', 'model/GetChildInfoApiKeysV3', 'model/GetChildInfoCredits', 'model/GetChildInfoStatistics', 'model/GetChildrenList', 'model/GetClient', 'model/GetContactCampaignStats', 'model/GetContactCampaignStatsClicked', 'model/GetContactCampaignStatsOpened', 'model/GetContactCampaignStatsTransacAttributes', 'model/GetContactCampaignStatsUnsubscriptions', 'model/GetContactDetails', 'model/GetContacts', 'model/GetDeviceBrowserStats', 'model/GetEmailCampaign', 'model/GetEmailCampaigns', 'model/GetEmailEventReport', 'model/GetEmailEventReportEvents', 'model/GetExtendedCampaignOverview', 'model/GetExtendedCampaignOverviewSender', 'model/GetExtendedCampaignStats', 'model/GetExtendedCampaignStatsGlobalStats', 'model/GetExtendedClient', 'model/GetExtendedClientAddress', 'model/GetExtendedContactDetails', 'model/GetExtendedContactDetailsStatistics', 'model/GetExtendedContactDetailsStatisticsClicked', 'model/GetExtendedContactDetailsStatisticsDelivered', 'model/GetExtendedContactDetailsStatisticsLinks', 'model/GetExtendedContactDetailsStatisticsMessagesSent', 'model/GetExtendedContactDetailsStatisticsOpened', 'model/GetExtendedContactDetailsStatisticsUnsubscriptions', 'model/GetExtendedContactDetailsStatisticsUnsubscriptionsAdminUnsubscription', 'model/GetExtendedContactDetailsStatisticsUnsubscriptionsUserUnsubscription', 'model/GetExtendedList', 'model/GetExtendedListCampaignStats', 'model/GetFolder', 'model/GetFolderLists', 'model/GetFolders', 'model/GetInboundEmailEvents', 'model/GetInboundEmailEventsByUuid', 'model/GetInboundEmailEventsByUuidAttachments', 'model/GetInboundEmailEventsByUuidLogs', 'model/GetInboundEmailEventsEvents', 'model/GetIp', 'model/GetIpFromSender', 'model/GetIps', 'model/GetIpsFromSender', 'model/GetList', 'model/GetLists', 'model/GetProcess', 'model/GetProcesses', 'model/GetReports', 'model/GetReportsReports', 'model/GetScheduledEmailByBatchId', 'model/GetScheduledEmailByBatchIdBatches', 'model/GetScheduledEmailByMessageId', 'model/GetSendersList', 'model/GetSendersListIps', 'model/GetSendersListSenders', 'model/GetSharedTemplateUrl', 'model/GetSmsCampaign', 'model/GetSmsCampaignOverview', 'model/GetSmsCampaignRecipients', 'model/GetSmsCampaignStats', 'model/GetSmsCampaigns', 'model/GetSmsEventReport', 'model/GetSmsEventReportEvents', 'model/GetSmtpTemplateOverview', 'model/GetSmtpTemplateOverviewSender', 'model/GetSmtpTemplates', 'model/GetSsoToken', 'model/GetStatsByBrowser', 'model/GetStatsByDevice', 'model/GetStatsByDomain', 'model/GetTransacAggregatedSmsReport', 'model/GetTransacBlockedContacts', 'model/GetTransacBlockedContactsContacts', 'model/GetTransacBlockedContactsReason', 'model/GetTransacEmailContent', 'model/GetTransacEmailContentEvents', 'model/GetTransacEmailsList', 'model/GetTransacEmailsListTransactionalEmails', 'model/GetTransacSmsReport', 'model/GetTransacSmsReportReports', 'model/GetWebhook', 'model/GetWebhooks', 'model/InlineResponse200', 'model/InlineResponse201', 'model/InlineResponse2011', 'model/ManageIp', 'model/MasterDetailsResponse', 'model/MasterDetailsResponseBillingInfo', 'model/MasterDetailsResponseBillingInfoAddress', 'model/MasterDetailsResponseBillingInfoName', 'model/MasterDetailsResponsePlanInfo', 'model/MasterDetailsResponsePlanInfoFeatures', 'model/Note', 'model/NoteData', 'model/NoteId', 'model/NoteList', 'model/Pipeline', 'model/PipelineStage', 'model/PostContactInfo', 'model/PostContactInfoContacts', 'model/PostSendFailed', 'model/PostSendSmsTestFailed', 'model/RemainingCreditModel', 'model/RemainingCreditModelChild', 'model/RemainingCreditModelReseller', 'model/RemoveContactFromList', 'model/RemoveCredits', 'model/RequestContactExport', 'model/RequestContactExportCustomContactFilter', 'model/RequestContactImport', 'model/RequestContactImportNewList', 'model/RequestSmsRecipientExport', 'model/ScheduleSmtpEmail', 'model/SendReport', 'model/SendReportEmail', 'model/SendSms', 'model/SendSmtpEmail', 'model/SendSmtpEmailAttachment', 'model/SendSmtpEmailBcc', 'model/SendSmtpEmailCc', 'model/SendSmtpEmailMessageVersions', 'model/SendSmtpEmailReplyTo', 'model/SendSmtpEmailReplyTo1', 'model/SendSmtpEmailSender', 'model/SendSmtpEmailTo', 'model/SendSmtpEmailTo1', 'model/SendTestEmail', 'model/SendTestSms', 'model/SendTransacSms', 'model/SsoTokenRequest', 'model/SubAccountDetailsResponse', 'model/SubAccountDetailsResponsePlanInfo', 'model/SubAccountDetailsResponsePlanInfoCredits', 'model/SubAccountDetailsResponsePlanInfoCreditsEmails', 'model/SubAccountDetailsResponsePlanInfoFeatures', 'model/SubAccountDetailsResponsePlanInfoFeaturesInbox', 'model/SubAccountDetailsResponsePlanInfoFeaturesLandingPage', 'model/SubAccountDetailsResponsePlanInfoFeaturesUsers', 'model/SubAccountUpdatePlanRequest', 'model/SubAccountUpdatePlanRequestCredits', 'model/SubAccountUpdatePlanRequestFeatures', 'model/SubAccountsResponse', 'model/SubAccountsResponseSubAccounts', 'model/Task', 'model/TaskList', 'model/TaskReminder', 'model/TaskTypes', 'model/UpdateAttribute', 'model/UpdateAttributeEnumeration', 'model/UpdateBatchContacts', 'model/UpdateBatchContactsContacts', 'model/UpdateBatchContactsModel', 'model/UpdateCampaignStatus', 'model/UpdateChild', 'model/UpdateChildAccountStatus', 'model/UpdateChildDomain', 'model/UpdateContact', 'model/UpdateEmailCampaign', 'model/UpdateEmailCampaignRecipients', 'model/UpdateEmailCampaignSender', 'model/UpdateList', 'model/UpdateSender', 'model/UpdateSmsCampaign', 'model/UpdateSmtpTemplate', 'model/UpdateSmtpTemplateSender', 'model/UpdateWebhook', 'model/UploadImageModel', 'model/UploadImageToGallery', 'api/AccountApi', 'api/CompaniesApi', 'api/ContactsApi', 'api/ConversationsApi', 'api/DealsApi', 'api/EmailCampaignsApi', 'api/FilesApi', 'api/InboundParsingApi', 'api/MasterAccountApi', 'api/NotesApi', 'api/ProcessApi', 'api/ResellerApi', 'api/SMSCampaignsApi', 'api/SendersApi', 'api/TasksApi', 'api/TransactionalEmailsApi', 'api/TransactionalSMSApi', 'api/WebhooksApi'], factory);
  } else if (typeof module === 'object' && module.exports) {
    // CommonJS-like environments that support module.exports, like Node.
    module.exports = factory(require('sib-api-v3-sdk/src/ApiClient'), require('sib-api-v3-sdk/src/model/AbTestCampaignResult'), require('sib-api-v3-sdk/src/model/AbTestCampaignResultClickedLinks'), require('sib-api-v3-sdk/src/model/AbTestCampaignResultStatistics'), require('sib-api-v3-sdk/src/model/AbTestVersionClicks'), require('sib-api-v3-sdk/src/model/AbTestVersionClicksInner'), require('sib-api-v3-sdk/src/model/AbTestVersionStats'), require('sib-api-v3-sdk/src/model/AddChildDomain'), require('sib-api-v3-sdk/src/model/AddContactToList'), require('sib-api-v3-sdk/src/model/AddCredits'), require('sib-api-v3-sdk/src/model/BlockDomain'), require('sib-api-v3-sdk/src/model/Body'), require('sib-api-v3-sdk/src/model/Body1'), require('sib-api-v3-sdk/src/model/Body10'), require('sib-api-v3-sdk/src/model/Body11'), require('sib-api-v3-sdk/src/model/Body12'), require('sib-api-v3-sdk/src/model/Body2'), require('sib-api-v3-sdk/src/model/Body3'), require('sib-api-v3-sdk/src/model/Body4'), require('sib-api-v3-sdk/src/model/Body5'), require('sib-api-v3-sdk/src/model/Body6'), require('sib-api-v3-sdk/src/model/Body7'), require('sib-api-v3-sdk/src/model/Body8'), require('sib-api-v3-sdk/src/model/Body9'), require('sib-api-v3-sdk/src/model/CompaniesList'), require('sib-api-v3-sdk/src/model/Company'), require('sib-api-v3-sdk/src/model/CompanyAttributes'), require('sib-api-v3-sdk/src/model/CompanyAttributesInner'), require('sib-api-v3-sdk/src/model/ConversationsMessage'), require('sib-api-v3-sdk/src/model/ConversationsMessageFile'), require('sib-api-v3-sdk/src/model/ConversationsMessageFileImageInfo'), require('sib-api-v3-sdk/src/model/CreateAttribute'), require('sib-api-v3-sdk/src/model/CreateAttributeEnumeration'), require('sib-api-v3-sdk/src/model/CreateChild'), require('sib-api-v3-sdk/src/model/CreateContact'), require('sib-api-v3-sdk/src/model/CreateDoiContact'), require('sib-api-v3-sdk/src/model/CreateEmailCampaign'), require('sib-api-v3-sdk/src/model/CreateEmailCampaignRecipients'), require('sib-api-v3-sdk/src/model/CreateEmailCampaignSender'), require('sib-api-v3-sdk/src/model/CreateList'), require('sib-api-v3-sdk/src/model/CreateModel'), require('sib-api-v3-sdk/src/model/CreateReseller'), require('sib-api-v3-sdk/src/model/CreateSender'), require('sib-api-v3-sdk/src/model/CreateSenderIps'), require('sib-api-v3-sdk/src/model/CreateSenderModel'), require('sib-api-v3-sdk/src/model/CreateSmsCampaign'), require('sib-api-v3-sdk/src/model/CreateSmsCampaignRecipients'), require('sib-api-v3-sdk/src/model/CreateSmtpEmail'), require('sib-api-v3-sdk/src/model/CreateSmtpTemplate'), require('sib-api-v3-sdk/src/model/CreateSmtpTemplateSender'), require('sib-api-v3-sdk/src/model/CreateSubAccount'), require('sib-api-v3-sdk/src/model/CreateSubAccountResponse'), require('sib-api-v3-sdk/src/model/CreateUpdateContactModel'), require('sib-api-v3-sdk/src/model/CreateUpdateFolder'), require('sib-api-v3-sdk/src/model/CreateWebhook'), require('sib-api-v3-sdk/src/model/CreatedProcessId'), require('sib-api-v3-sdk/src/model/Deal'), require('sib-api-v3-sdk/src/model/DealAttributes'), require('sib-api-v3-sdk/src/model/DealAttributesInner'), require('sib-api-v3-sdk/src/model/DealsList'), require('sib-api-v3-sdk/src/model/DeleteHardbounces'), require('sib-api-v3-sdk/src/model/EmailExportRecipients'), require('sib-api-v3-sdk/src/model/ErrorModel'), require('sib-api-v3-sdk/src/model/FileData'), require('sib-api-v3-sdk/src/model/FileDownloadableLink'), require('sib-api-v3-sdk/src/model/FileList'), require('sib-api-v3-sdk/src/model/GetAccount'), require('sib-api-v3-sdk/src/model/GetAccountMarketingAutomation'), require('sib-api-v3-sdk/src/model/GetAccountPlan'), require('sib-api-v3-sdk/src/model/GetAccountRelay'), require('sib-api-v3-sdk/src/model/GetAccountRelayData'), require('sib-api-v3-sdk/src/model/GetAggregatedReport'), require('sib-api-v3-sdk/src/model/GetAttributes'), require('sib-api-v3-sdk/src/model/GetAttributesAttributes'), require('sib-api-v3-sdk/src/model/GetAttributesEnumeration'), require('sib-api-v3-sdk/src/model/GetBlockedDomains'), require('sib-api-v3-sdk/src/model/GetCampaignOverview'), require('sib-api-v3-sdk/src/model/GetCampaignRecipients'), require('sib-api-v3-sdk/src/model/GetCampaignStats'), require('sib-api-v3-sdk/src/model/GetChildAccountCreationStatus'), require('sib-api-v3-sdk/src/model/GetChildDomain'), require('sib-api-v3-sdk/src/model/GetChildDomains'), require('sib-api-v3-sdk/src/model/GetChildInfo'), require('sib-api-v3-sdk/src/model/GetChildInfoApiKeys'), require('sib-api-v3-sdk/src/model/GetChildInfoApiKeysV2'), require('sib-api-v3-sdk/src/model/GetChildInfoApiKeysV3'), require('sib-api-v3-sdk/src/model/GetChildInfoCredits'), require('sib-api-v3-sdk/src/model/GetChildInfoStatistics'), require('sib-api-v3-sdk/src/model/GetChildrenList'), require('sib-api-v3-sdk/src/model/GetClient'), require('sib-api-v3-sdk/src/model/GetContactCampaignStats'), require('sib-api-v3-sdk/src/model/GetContactCampaignStatsClicked'), require('sib-api-v3-sdk/src/model/GetContactCampaignStatsOpened'), require('sib-api-v3-sdk/src/model/GetContactCampaignStatsTransacAttributes'), require('sib-api-v3-sdk/src/model/GetContactCampaignStatsUnsubscriptions'), require('sib-api-v3-sdk/src/model/GetContactDetails'), require('sib-api-v3-sdk/src/model/GetContacts'), require('sib-api-v3-sdk/src/model/GetDeviceBrowserStats'), require('sib-api-v3-sdk/src/model/GetEmailCampaign'), require('sib-api-v3-sdk/src/model/GetEmailCampaigns'), require('sib-api-v3-sdk/src/model/GetEmailEventReport'), require('sib-api-v3-sdk/src/model/GetEmailEventReportEvents'), require('sib-api-v3-sdk/src/model/GetExtendedCampaignOverview'), require('sib-api-v3-sdk/src/model/GetExtendedCampaignOverviewSender'), require('sib-api-v3-sdk/src/model/GetExtendedCampaignStats'), require('sib-api-v3-sdk/src/model/GetExtendedCampaignStatsGlobalStats'), require('sib-api-v3-sdk/src/model/GetExtendedClient'), require('sib-api-v3-sdk/src/model/GetExtendedClientAddress'), require('sib-api-v3-sdk/src/model/GetExtendedContactDetails'), require('sib-api-v3-sdk/src/model/GetExtendedContactDetailsStatistics'), require('sib-api-v3-sdk/src/model/GetExtendedContactDetailsStatisticsClicked'), require('sib-api-v3-sdk/src/model/GetExtendedContactDetailsStatisticsDelivered'), require('sib-api-v3-sdk/src/model/GetExtendedContactDetailsStatisticsLinks'), require('sib-api-v3-sdk/src/model/GetExtendedContactDetailsStatisticsMessagesSent'), require('sib-api-v3-sdk/src/model/GetExtendedContactDetailsStatisticsOpened'), require('sib-api-v3-sdk/src/model/GetExtendedContactDetailsStatisticsUnsubscriptions'), require('sib-api-v3-sdk/src/model/GetExtendedContactDetailsStatisticsUnsubscriptionsAdminUnsubscription'), require('sib-api-v3-sdk/src/model/GetExtendedContactDetailsStatisticsUnsubscriptionsUserUnsubscription'), require('sib-api-v3-sdk/src/model/GetExtendedList'), require('sib-api-v3-sdk/src/model/GetExtendedListCampaignStats'), require('sib-api-v3-sdk/src/model/GetFolder'), require('sib-api-v3-sdk/src/model/GetFolderLists'), require('sib-api-v3-sdk/src/model/GetFolders'), require('sib-api-v3-sdk/src/model/GetInboundEmailEvents'), require('sib-api-v3-sdk/src/model/GetInboundEmailEventsByUuid'), require('sib-api-v3-sdk/src/model/GetInboundEmailEventsByUuidAttachments'), require('sib-api-v3-sdk/src/model/GetInboundEmailEventsByUuidLogs'), require('sib-api-v3-sdk/src/model/GetInboundEmailEventsEvents'), require('sib-api-v3-sdk/src/model/GetIp'), require('sib-api-v3-sdk/src/model/GetIpFromSender'), require('sib-api-v3-sdk/src/model/GetIps'), require('sib-api-v3-sdk/src/model/GetIpsFromSender'), require('sib-api-v3-sdk/src/model/GetList'), require('sib-api-v3-sdk/src/model/GetLists'), require('sib-api-v3-sdk/src/model/GetProcess'), require('sib-api-v3-sdk/src/model/GetProcesses'), require('sib-api-v3-sdk/src/model/GetReports'), require('sib-api-v3-sdk/src/model/GetReportsReports'), require('sib-api-v3-sdk/src/model/GetScheduledEmailByBatchId'), require('sib-api-v3-sdk/src/model/GetScheduledEmailByBatchIdBatches'), require('sib-api-v3-sdk/src/model/GetScheduledEmailByMessageId'), require('sib-api-v3-sdk/src/model/GetSendersList'), require('sib-api-v3-sdk/src/model/GetSendersListIps'), require('sib-api-v3-sdk/src/model/GetSendersListSenders'), require('sib-api-v3-sdk/src/model/GetSharedTemplateUrl'), require('sib-api-v3-sdk/src/model/GetSmsCampaign'), require('sib-api-v3-sdk/src/model/GetSmsCampaignOverview'), require('sib-api-v3-sdk/src/model/GetSmsCampaignRecipients'), require('sib-api-v3-sdk/src/model/GetSmsCampaignStats'), require('sib-api-v3-sdk/src/model/GetSmsCampaigns'), require('sib-api-v3-sdk/src/model/GetSmsEventReport'), require('sib-api-v3-sdk/src/model/GetSmsEventReportEvents'), require('sib-api-v3-sdk/src/model/GetSmtpTemplateOverview'), require('sib-api-v3-sdk/src/model/GetSmtpTemplateOverviewSender'), require('sib-api-v3-sdk/src/model/GetSmtpTemplates'), require('sib-api-v3-sdk/src/model/GetSsoToken'), require('sib-api-v3-sdk/src/model/GetStatsByBrowser'), require('sib-api-v3-sdk/src/model/GetStatsByDevice'), require('sib-api-v3-sdk/src/model/GetStatsByDomain'), require('sib-api-v3-sdk/src/model/GetTransacAggregatedSmsReport'), require('sib-api-v3-sdk/src/model/GetTransacBlockedContacts'), require('sib-api-v3-sdk/src/model/GetTransacBlockedContactsContacts'), require('sib-api-v3-sdk/src/model/GetTransacBlockedContactsReason'), require('sib-api-v3-sdk/src/model/GetTransacEmailContent'), require('sib-api-v3-sdk/src/model/GetTransacEmailContentEvents'), require('sib-api-v3-sdk/src/model/GetTransacEmailsList'), require('sib-api-v3-sdk/src/model/GetTransacEmailsListTransactionalEmails'), require('sib-api-v3-sdk/src/model/GetTransacSmsReport'), require('sib-api-v3-sdk/src/model/GetTransacSmsReportReports'), require('sib-api-v3-sdk/src/model/GetWebhook'), require('sib-api-v3-sdk/src/model/GetWebhooks'), require('sib-api-v3-sdk/src/model/InlineResponse200'), require('sib-api-v3-sdk/src/model/InlineResponse201'), require('sib-api-v3-sdk/src/model/InlineResponse2011'), require('sib-api-v3-sdk/src/model/ManageIp'), require('sib-api-v3-sdk/src/model/MasterDetailsResponse'), require('sib-api-v3-sdk/src/model/MasterDetailsResponseBillingInfo'), require('sib-api-v3-sdk/src/model/MasterDetailsResponseBillingInfoAddress'), require('sib-api-v3-sdk/src/model/MasterDetailsResponseBillingInfoName'), require('sib-api-v3-sdk/src/model/MasterDetailsResponsePlanInfo'), require('sib-api-v3-sdk/src/model/MasterDetailsResponsePlanInfoFeatures'), require('sib-api-v3-sdk/src/model/Note'), require('sib-api-v3-sdk/src/model/NoteData'), require('sib-api-v3-sdk/src/model/NoteId'), require('sib-api-v3-sdk/src/model/NoteList'), require('sib-api-v3-sdk/src/model/Pipeline'), require('sib-api-v3-sdk/src/model/PipelineStage'), require('sib-api-v3-sdk/src/model/PostContactInfo'), require('sib-api-v3-sdk/src/model/PostContactInfoContacts'), require('sib-api-v3-sdk/src/model/PostSendFailed'), require('sib-api-v3-sdk/src/model/PostSendSmsTestFailed'), require('sib-api-v3-sdk/src/model/RemainingCreditModel'), require('sib-api-v3-sdk/src/model/RemainingCreditModelChild'), require('sib-api-v3-sdk/src/model/RemainingCreditModelReseller'), require('sib-api-v3-sdk/src/model/RemoveContactFromList'), require('sib-api-v3-sdk/src/model/RemoveCredits'), require('sib-api-v3-sdk/src/model/RequestContactExport'), require('sib-api-v3-sdk/src/model/RequestContactExportCustomContactFilter'), require('sib-api-v3-sdk/src/model/RequestContactImport'), require('sib-api-v3-sdk/src/model/RequestContactImportNewList'), require('sib-api-v3-sdk/src/model/RequestSmsRecipientExport'), require('sib-api-v3-sdk/src/model/ScheduleSmtpEmail'), require('sib-api-v3-sdk/src/model/SendReport'), require('sib-api-v3-sdk/src/model/SendReportEmail'), require('sib-api-v3-sdk/src/model/SendSms'), require('sib-api-v3-sdk/src/model/SendSmtpEmail'), require('sib-api-v3-sdk/src/model/SendSmtpEmailAttachment'), require('sib-api-v3-sdk/src/model/SendSmtpEmailBcc'), require('sib-api-v3-sdk/src/model/SendSmtpEmailCc'), require('sib-api-v3-sdk/src/model/SendSmtpEmailMessageVersions'), require('sib-api-v3-sdk/src/model/SendSmtpEmailReplyTo'), require('sib-api-v3-sdk/src/model/SendSmtpEmailReplyTo1'), require('sib-api-v3-sdk/src/model/SendSmtpEmailSender'), require('sib-api-v3-sdk/src/model/SendSmtpEmailTo'), require('sib-api-v3-sdk/src/model/SendSmtpEmailTo1'), require('sib-api-v3-sdk/src/model/SendTestEmail'), require('sib-api-v3-sdk/src/model/SendTestSms'), require('sib-api-v3-sdk/src/model/SendTransacSms'), require('sib-api-v3-sdk/src/model/SsoTokenRequest'), require('sib-api-v3-sdk/src/model/SubAccountDetailsResponse'), require('sib-api-v3-sdk/src/model/SubAccountDetailsResponsePlanInfo'), require('sib-api-v3-sdk/src/model/SubAccountDetailsResponsePlanInfoCredits'), require('sib-api-v3-sdk/src/model/SubAccountDetailsResponsePlanInfoCreditsEmails'), require('sib-api-v3-sdk/src/model/SubAccountDetailsResponsePlanInfoFeatures'), require('sib-api-v3-sdk/src/model/SubAccountDetailsResponsePlanInfoFeaturesInbox'), require('sib-api-v3-sdk/src/model/SubAccountDetailsResponsePlanInfoFeaturesLandingPage'), require('sib-api-v3-sdk/src/model/SubAccountDetailsResponsePlanInfoFeaturesUsers'), require('sib-api-v3-sdk/src/model/SubAccountUpdatePlanRequest'), require('sib-api-v3-sdk/src/model/SubAccountUpdatePlanRequestCredits'), require('sib-api-v3-sdk/src/model/SubAccountUpdatePlanRequestFeatures'), require('sib-api-v3-sdk/src/model/SubAccountsResponse'), require('sib-api-v3-sdk/src/model/SubAccountsResponseSubAccounts'), require('sib-api-v3-sdk/src/model/Task'), require('sib-api-v3-sdk/src/model/TaskList'), require('sib-api-v3-sdk/src/model/TaskReminder'), require('sib-api-v3-sdk/src/model/TaskTypes'), require('sib-api-v3-sdk/src/model/UpdateAttribute'), require('sib-api-v3-sdk/src/model/UpdateAttributeEnumeration'), require('sib-api-v3-sdk/src/model/UpdateBatchContacts'), require('sib-api-v3-sdk/src/model/UpdateBatchContactsContacts'), require('sib-api-v3-sdk/src/model/UpdateBatchContactsModel'), require('sib-api-v3-sdk/src/model/UpdateCampaignStatus'), require('sib-api-v3-sdk/src/model/UpdateChild'), require('sib-api-v3-sdk/src/model/UpdateChildAccountStatus'), require('sib-api-v3-sdk/src/model/UpdateChildDomain'), require('sib-api-v3-sdk/src/model/UpdateContact'), require('sib-api-v3-sdk/src/model/UpdateEmailCampaign'), require('sib-api-v3-sdk/src/model/UpdateEmailCampaignRecipients'), require('sib-api-v3-sdk/src/model/UpdateEmailCampaignSender'), require('sib-api-v3-sdk/src/model/UpdateList'), require('sib-api-v3-sdk/src/model/UpdateSender'), require('sib-api-v3-sdk/src/model/UpdateSmsCampaign'), require('sib-api-v3-sdk/src/model/UpdateSmtpTemplate'), require('sib-api-v3-sdk/src/model/UpdateSmtpTemplateSender'), require('sib-api-v3-sdk/src/model/UpdateWebhook'), require('sib-api-v3-sdk/src/model/UploadImageModel'), require('sib-api-v3-sdk/src/model/UploadImageToGallery'), require('sib-api-v3-sdk/src/api/AccountApi'), require('sib-api-v3-sdk/src/api/CompaniesApi'), require('sib-api-v3-sdk/src/api/ContactsApi'), require('sib-api-v3-sdk/src/api/ConversationsApi'), require('sib-api-v3-sdk/src/api/DealsApi'), require('sib-api-v3-sdk/src/api/EmailCampaignsApi'), require('sib-api-v3-sdk/src/api/FilesApi'), require('sib-api-v3-sdk/src/api/InboundParsingApi'), require('sib-api-v3-sdk/src/api/MasterAccountApi'), require('sib-api-v3-sdk/src/api/NotesApi'), require('sib-api-v3-sdk/src/api/ProcessApi'), require('sib-api-v3-sdk/src/api/ResellerApi'), require('sib-api-v3-sdk/src/api/SMSCampaignsApi'), require('sib-api-v3-sdk/src/api/SendersApi'), require('sib-api-v3-sdk/src/api/TasksApi'), require('sib-api-v3-sdk/src/api/TransactionalEmailsApi'), require('sib-api-v3-sdk/src/api/TransactionalSMSApi'), require('sib-api-v3-sdk/src/api/WebhooksApi'));
  }
}(function(ApiClient, AbTestCampaignResult, AbTestCampaignResultClickedLinks, AbTestCampaignResultStatistics, AbTestVersionClicks, AbTestVersionClicksInner, AbTestVersionStats, AddChildDomain, AddContactToList, AddCredits, BlockDomain, Body, Body1, Body10, Body11, Body12, Body2, Body3, Body4, Body5, Body6, Body7, Body8, Body9, CompaniesList, Company, CompanyAttributes, CompanyAttributesInner, ConversationsMessage, ConversationsMessageFile, ConversationsMessageFileImageInfo, CreateAttribute, CreateAttributeEnumeration, CreateChild, CreateContact, CreateDoiContact, CreateEmailCampaign, CreateEmailCampaignRecipients, CreateEmailCampaignSender, CreateList, CreateModel, CreateReseller, CreateSender, CreateSenderIps, CreateSenderModel, CreateSmsCampaign, CreateSmsCampaignRecipients, CreateSmtpEmail, CreateSmtpTemplate, CreateSmtpTemplateSender, CreateSubAccount, CreateSubAccountResponse, CreateUpdateContactModel, CreateUpdateFolder, CreateWebhook, CreatedProcessId, Deal, DealAttributes, DealAttributesInner, DealsList, DeleteHardbounces, EmailExportRecipients, ErrorModel, FileData, FileDownloadableLink, FileList, GetAccount, GetAccountMarketingAutomation, GetAccountPlan, GetAccountRelay, GetAccountRelayData, GetAggregatedReport, GetAttributes, GetAttributesAttributes, GetAttributesEnumeration, GetBlockedDomains, GetCampaignOverview, GetCampaignRecipients, GetCampaignStats, GetChildAccountCreationStatus, GetChildDomain, GetChildDomains, GetChildInfo, GetChildInfoApiKeys, GetChildInfoApiKeysV2, GetChildInfoApiKeysV3, GetChildInfoCredits, GetChildInfoStatistics, GetChildrenList, GetClient, GetContactCampaignStats, GetContactCampaignStatsClicked, GetContactCampaignStatsOpened, GetContactCampaignStatsTransacAttributes, GetContactCampaignStatsUnsubscriptions, GetContactDetails, GetContacts, GetDeviceBrowserStats, GetEmailCampaign, GetEmailCampaigns, GetEmailEventReport, GetEmailEventReportEvents, GetExtendedCampaignOverview, GetExtendedCampaignOverviewSender, GetExtendedCampaignStats, GetExtendedCampaignStatsGlobalStats, GetExtendedClient, GetExtendedClientAddress, GetExtendedContactDetails, GetExtendedContactDetailsStatistics, GetExtendedContactDetailsStatisticsClicked, GetExtendedContactDetailsStatisticsDelivered, GetExtendedContactDetailsStatisticsLinks, GetExtendedContactDetailsStatisticsMessagesSent, GetExtendedContactDetailsStatisticsOpened, GetExtendedContactDetailsStatisticsUnsubscriptions, GetExtendedContactDetailsStatisticsUnsubscriptionsAdminUnsubscription, GetExtendedContactDetailsStatisticsUnsubscriptionsUserUnsubscription, GetExtendedList, GetExtendedListCampaignStats, GetFolder, GetFolderLists, GetFolders, GetInboundEmailEvents, GetInboundEmailEventsByUuid, GetInboundEmailEventsByUuidAttachments, GetInboundEmailEventsByUuidLogs, GetInboundEmailEventsEvents, GetIp, GetIpFromSender, GetIps, GetIpsFromSender, GetList, GetLists, GetProcess, GetProcesses, GetReports, GetReportsReports, GetScheduledEmailByBatchId, GetScheduledEmailByBatchIdBatches, GetScheduledEmailByMessageId, GetSendersList, GetSendersListIps, GetSendersListSenders, GetSharedTemplateUrl, GetSmsCampaign, GetSmsCampaignOverview, GetSmsCampaignRecipients, GetSmsCampaignStats, GetSmsCampaigns, GetSmsEventReport, GetSmsEventReportEvents, GetSmtpTemplateOverview, GetSmtpTemplateOverviewSender, GetSmtpTemplates, GetSsoToken, GetStatsByBrowser, GetStatsByDevice, GetStatsByDomain, GetTransacAggregatedSmsReport, GetTransacBlockedContacts, GetTransacBlockedContactsContacts, GetTransacBlockedContactsReason, GetTransacEmailContent, GetTransacEmailContentEvents, GetTransacEmailsList, GetTransacEmailsListTransactionalEmails, GetTransacSmsReport, GetTransacSmsReportReports, GetWebhook, GetWebhooks, InlineResponse200, InlineResponse201, InlineResponse2011, ManageIp, MasterDetailsResponse, MasterDetailsResponseBillingInfo, MasterDetailsResponseBillingInfoAddress, MasterDetailsResponseBillingInfoName, MasterDetailsResponsePlanInfo, MasterDetailsResponsePlanInfoFeatures, Note, NoteData, NoteId, NoteList, Pipeline, PipelineStage, PostContactInfo, PostContactInfoContacts, PostSendFailed, PostSendSmsTestFailed, RemainingCreditModel, RemainingCreditModelChild, RemainingCreditModelReseller, RemoveContactFromList, RemoveCredits, RequestContactExport, RequestContactExportCustomContactFilter, RequestContactImport, RequestContactImportNewList, RequestSmsRecipientExport, ScheduleSmtpEmail, SendReport, SendReportEmail, SendSms, SendSmtpEmail, SendSmtpEmailAttachment, SendSmtpEmailBcc, SendSmtpEmailCc, SendSmtpEmailMessageVersions, SendSmtpEmailReplyTo, SendSmtpEmailReplyTo1, SendSmtpEmailSender, SendSmtpEmailTo, SendSmtpEmailTo1, SendTestEmail, SendTestSms, SendTransacSms, SsoTokenRequest, SubAccountDetailsResponse, SubAccountDetailsResponsePlanInfo, SubAccountDetailsResponsePlanInfoCredits, SubAccountDetailsResponsePlanInfoCreditsEmails, SubAccountDetailsResponsePlanInfoFeatures, SubAccountDetailsResponsePlanInfoFeaturesInbox, SubAccountDetailsResponsePlanInfoFeaturesLandingPage, SubAccountDetailsResponsePlanInfoFeaturesUsers, SubAccountUpdatePlanRequest, SubAccountUpdatePlanRequestCredits, SubAccountUpdatePlanRequestFeatures, SubAccountsResponse, SubAccountsResponseSubAccounts, Task, TaskList, TaskReminder, TaskTypes, UpdateAttribute, UpdateAttributeEnumeration, UpdateBatchContacts, UpdateBatchContactsContacts, UpdateBatchContactsModel, UpdateCampaignStatus, UpdateChild, UpdateChildAccountStatus, UpdateChildDomain, UpdateContact, UpdateEmailCampaign, UpdateEmailCampaignRecipients, UpdateEmailCampaignSender, UpdateList, UpdateSender, UpdateSmsCampaign, UpdateSmtpTemplate, UpdateSmtpTemplateSender, UpdateWebhook, UploadImageModel, UploadImageToGallery, AccountApi, CompaniesApi, ContactsApi, ConversationsApi, DealsApi, EmailCampaignsApi, FilesApi, InboundParsingApi, MasterAccountApi, NotesApi, ProcessApi, ResellerApi, SMSCampaignsApi, SendersApi, TasksApi, TransactionalEmailsApi, TransactionalSMSApi, WebhooksApi) {
  'use strict';

  /**
   * SendinBlue_provide_a_RESTFul_API_that_can_be_used_with_any_languages__With_this_API_you_will_be_able_to_____Manage_your_campaigns_and_get_the_statistics____Manage_your_contacts____Send_transactional_Emails_and_SMS____and_much_more___You_can_download_our_wrappers_at_httpsgithub_comorgssendinbluePossible_responses___Code__Message____________________________________200___OK__Successful_Request_____201___OK__Successful_Creation____202___OK__Request_accepted____204___OK__Successful_UpdateDeletion_____400___Error__Bad_Request_____401___Error__Authentication_Needed_____402___Error__Not_enough_credit_plan_upgrade_needed_____403___Error__Permission_denied_____404___Error__Object_does_not_exist____405___Error__Method_not_allowed_____406___Error__Not_Acceptable__.<br>
   * The <code>index</code> module provides access to constructors for all the classes which comprise the public API.
   * <p>
   * An AMD (recommended!) or CommonJS application will generally do something equivalent to the following:
   * <pre>
   * var SibApiV3Sdk = require('index'); // See note below*.
   * var xxxSvc = new SibApiV3Sdk.XxxApi(); // Allocate the API class we're going to use.
   * var yyyModel = new SibApiV3Sdk.Yyy(); // Construct a model instance.
   * yyyModel.someProperty = 'someValue';
   * ...
   * var zzz = xxxSvc.doSomething(yyyModel); // Invoke the service.
   * ...
   * </pre>
   * <em>*NOTE: For a top-level AMD script, use require(['index'], function(){...})
   * and put the application logic within the callback function.</em>
   * </p>
   * <p>
   * A non-AMD browser application (discouraged) might do something like this:
   * <pre>
   * var xxxSvc = new SibApiV3Sdk.XxxApi(); // Allocate the API class we're going to use.
   * var yyy = new SibApiV3Sdk.Yyy(); // Construct a model instance.
   * yyyModel.someProperty = 'someValue';
   * ...
   * var zzz = xxxSvc.doSomething(yyyModel); // Invoke the service.
   * ...
   * </pre>
   * </p>
   * @module index
   * @version 8.5.0
   */
  var exports = {
    /**
     * The ApiClient constructor.
     * @property {module:ApiClient}
     */
    ApiClient: ApiClient,
    /**
     * The AbTestCampaignResult model constructor.
     * @property {module:model/AbTestCampaignResult}
     */
    AbTestCampaignResult: AbTestCampaignResult,
    /**
     * The AbTestCampaignResultClickedLinks model constructor.
     * @property {module:model/AbTestCampaignResultClickedLinks}
     */
    AbTestCampaignResultClickedLinks: AbTestCampaignResultClickedLinks,
    /**
     * The AbTestCampaignResultStatistics model constructor.
     * @property {module:model/AbTestCampaignResultStatistics}
     */
    AbTestCampaignResultStatistics: AbTestCampaignResultStatistics,
    /**
     * The AbTestVersionClicks model constructor.
     * @property {module:model/AbTestVersionClicks}
     */
    AbTestVersionClicks: AbTestVersionClicks,
    /**
     * The AbTestVersionClicksInner model constructor.
     * @property {module:model/AbTestVersionClicksInner}
     */
    AbTestVersionClicksInner: AbTestVersionClicksInner,
    /**
     * The AbTestVersionStats model constructor.
     * @property {module:model/AbTestVersionStats}
     */
    AbTestVersionStats: AbTestVersionStats,
    /**
     * The AddChildDomain model constructor.
     * @property {module:model/AddChildDomain}
     */
    AddChildDomain: AddChildDomain,
    /**
     * The AddContactToList model constructor.
     * @property {module:model/AddContactToList}
     */
    AddContactToList: AddContactToList,
    /**
     * The AddCredits model constructor.
     * @property {module:model/AddCredits}
     */
    AddCredits: AddCredits,
    /**
     * The BlockDomain model constructor.
     * @property {module:model/BlockDomain}
     */
    BlockDomain: BlockDomain,
    /**
     * The Body model constructor.
     * @property {module:model/Body}
     */
    Body: Body,
    /**
     * The Body1 model constructor.
     * @property {module:model/Body1}
     */
    Body1: Body1,
    /**
     * The Body10 model constructor.
     * @property {module:model/Body10}
     */
    Body10: Body10,
    /**
     * The Body11 model constructor.
     * @property {module:model/Body11}
     */
    Body11: Body11,
    /**
     * The Body12 model constructor.
     * @property {module:model/Body12}
     */
    Body12: Body12,
    /**
     * The Body2 model constructor.
     * @property {module:model/Body2}
     */
    Body2: Body2,
    /**
     * The Body3 model constructor.
     * @property {module:model/Body3}
     */
    Body3: Body3,
    /**
     * The Body4 model constructor.
     * @property {module:model/Body4}
     */
    Body4: Body4,
    /**
     * The Body5 model constructor.
     * @property {module:model/Body5}
     */
    Body5: Body5,
    /**
     * The Body6 model constructor.
     * @property {module:model/Body6}
     */
    Body6: Body6,
    /**
     * The Body7 model constructor.
     * @property {module:model/Body7}
     */
    Body7: Body7,
    /**
     * The Body8 model constructor.
     * @property {module:model/Body8}
     */
    Body8: Body8,
    /**
     * The Body9 model constructor.
     * @property {module:model/Body9}
     */
    Body9: Body9,
    /**
     * The CompaniesList model constructor.
     * @property {module:model/CompaniesList}
     */
    CompaniesList: CompaniesList,
    /**
     * The Company model constructor.
     * @property {module:model/Company}
     */
    Company: Company,
    /**
     * The CompanyAttributes model constructor.
     * @property {module:model/CompanyAttributes}
     */
    CompanyAttributes: CompanyAttributes,
    /**
     * The CompanyAttributesInner model constructor.
     * @property {module:model/CompanyAttributesInner}
     */
    CompanyAttributesInner: CompanyAttributesInner,
    /**
     * The ConversationsMessage model constructor.
     * @property {module:model/ConversationsMessage}
     */
    ConversationsMessage: ConversationsMessage,
    /**
     * The ConversationsMessageFile model constructor.
     * @property {module:model/ConversationsMessageFile}
     */
    ConversationsMessageFile: ConversationsMessageFile,
    /**
     * The ConversationsMessageFileImageInfo model constructor.
     * @property {module:model/ConversationsMessageFileImageInfo}
     */
    ConversationsMessageFileImageInfo: ConversationsMessageFileImageInfo,
    /**
     * The CreateAttribute model constructor.
     * @property {module:model/CreateAttribute}
     */
    CreateAttribute: CreateAttribute,
    /**
     * The CreateAttributeEnumeration model constructor.
     * @property {module:model/CreateAttributeEnumeration}
     */
    CreateAttributeEnumeration: CreateAttributeEnumeration,
    /**
     * The CreateChild model constructor.
     * @property {module:model/CreateChild}
     */
    CreateChild: CreateChild,
    /**
     * The CreateContact model constructor.
     * @property {module:model/CreateContact}
     */
    CreateContact: CreateContact,
    /**
     * The CreateDoiContact model constructor.
     * @property {module:model/CreateDoiContact}
     */
    CreateDoiContact: CreateDoiContact,
    /**
     * The CreateEmailCampaign model constructor.
     * @property {module:model/CreateEmailCampaign}
     */
    CreateEmailCampaign: CreateEmailCampaign,
    /**
     * The CreateEmailCampaignRecipients model constructor.
     * @property {module:model/CreateEmailCampaignRecipients}
     */
    CreateEmailCampaignRecipients: CreateEmailCampaignRecipients,
    /**
     * The CreateEmailCampaignSender model constructor.
     * @property {module:model/CreateEmailCampaignSender}
     */
    CreateEmailCampaignSender: CreateEmailCampaignSender,
    /**
     * The CreateList model constructor.
     * @property {module:model/CreateList}
     */
    CreateList: CreateList,
    /**
     * The CreateModel model constructor.
     * @property {module:model/CreateModel}
     */
    CreateModel: CreateModel,
    /**
     * The CreateReseller model constructor.
     * @property {module:model/CreateReseller}
     */
    CreateReseller: CreateReseller,
    /**
     * The CreateSender model constructor.
     * @property {module:model/CreateSender}
     */
    CreateSender: CreateSender,
    /**
     * The CreateSenderIps model constructor.
     * @property {module:model/CreateSenderIps}
     */
    CreateSenderIps: CreateSenderIps,
    /**
     * The CreateSenderModel model constructor.
     * @property {module:model/CreateSenderModel}
     */
    CreateSenderModel: CreateSenderModel,
    /**
     * The CreateSmsCampaign model constructor.
     * @property {module:model/CreateSmsCampaign}
     */
    CreateSmsCampaign: CreateSmsCampaign,
    /**
     * The CreateSmsCampaignRecipients model constructor.
     * @property {module:model/CreateSmsCampaignRecipients}
     */
    CreateSmsCampaignRecipients: CreateSmsCampaignRecipients,
    /**
     * The CreateSmtpEmail model constructor.
     * @property {module:model/CreateSmtpEmail}
     */
    CreateSmtpEmail: CreateSmtpEmail,
    /**
     * The CreateSmtpTemplate model constructor.
     * @property {module:model/CreateSmtpTemplate}
     */
    CreateSmtpTemplate: CreateSmtpTemplate,
    /**
     * The CreateSmtpTemplateSender model constructor.
     * @property {module:model/CreateSmtpTemplateSender}
     */
    CreateSmtpTemplateSender: CreateSmtpTemplateSender,
    /**
     * The CreateSubAccount model constructor.
     * @property {module:model/CreateSubAccount}
     */
    CreateSubAccount: CreateSubAccount,
    /**
     * The CreateSubAccountResponse model constructor.
     * @property {module:model/CreateSubAccountResponse}
     */
    CreateSubAccountResponse: CreateSubAccountResponse,
    /**
     * The CreateUpdateContactModel model constructor.
     * @property {module:model/CreateUpdateContactModel}
     */
    CreateUpdateContactModel: CreateUpdateContactModel,
    /**
     * The CreateUpdateFolder model constructor.
     * @property {module:model/CreateUpdateFolder}
     */
    CreateUpdateFolder: CreateUpdateFolder,
    /**
     * The CreateWebhook model constructor.
     * @property {module:model/CreateWebhook}
     */
    CreateWebhook: CreateWebhook,
    /**
     * The CreatedProcessId model constructor.
     * @property {module:model/CreatedProcessId}
     */
    CreatedProcessId: CreatedProcessId,
    /**
     * The Deal model constructor.
     * @property {module:model/Deal}
     */
    Deal: Deal,
    /**
     * The DealAttributes model constructor.
     * @property {module:model/DealAttributes}
     */
    DealAttributes: DealAttributes,
    /**
     * The DealAttributesInner model constructor.
     * @property {module:model/DealAttributesInner}
     */
    DealAttributesInner: DealAttributesInner,
    /**
     * The DealsList model constructor.
     * @property {module:model/DealsList}
     */
    DealsList: DealsList,
    /**
     * The DeleteHardbounces model constructor.
     * @property {module:model/DeleteHardbounces}
     */
    DeleteHardbounces: DeleteHardbounces,
    /**
     * The EmailExportRecipients model constructor.
     * @property {module:model/EmailExportRecipients}
     */
    EmailExportRecipients: EmailExportRecipients,
    /**
     * The ErrorModel model constructor.
     * @property {module:model/ErrorModel}
     */
    ErrorModel: ErrorModel,
    /**
     * The FileData model constructor.
     * @property {module:model/FileData}
     */
    FileData: FileData,
    /**
     * The FileDownloadableLink model constructor.
     * @property {module:model/FileDownloadableLink}
     */
    FileDownloadableLink: FileDownloadableLink,
    /**
     * The FileList model constructor.
     * @property {module:model/FileList}
     */
    FileList: FileList,
    /**
     * The GetAccount model constructor.
     * @property {module:model/GetAccount}
     */
    GetAccount: GetAccount,
    /**
     * The GetAccountMarketingAutomation model constructor.
     * @property {module:model/GetAccountMarketingAutomation}
     */
    GetAccountMarketingAutomation: GetAccountMarketingAutomation,
    /**
     * The GetAccountPlan model constructor.
     * @property {module:model/GetAccountPlan}
     */
    GetAccountPlan: GetAccountPlan,
    /**
     * The GetAccountRelay model constructor.
     * @property {module:model/GetAccountRelay}
     */
    GetAccountRelay: GetAccountRelay,
    /**
     * The GetAccountRelayData model constructor.
     * @property {module:model/GetAccountRelayData}
     */
    GetAccountRelayData: GetAccountRelayData,
    /**
     * The GetAggregatedReport model constructor.
     * @property {module:model/GetAggregatedReport}
     */
    GetAggregatedReport: GetAggregatedReport,
    /**
     * The GetAttributes model constructor.
     * @property {module:model/GetAttributes}
     */
    GetAttributes: GetAttributes,
    /**
     * The GetAttributesAttributes model constructor.
     * @property {module:model/GetAttributesAttributes}
     */
    GetAttributesAttributes: GetAttributesAttributes,
    /**
     * The GetAttributesEnumeration model constructor.
     * @property {module:model/GetAttributesEnumeration}
     */
    GetAttributesEnumeration: GetAttributesEnumeration,
    /**
     * The GetBlockedDomains model constructor.
     * @property {module:model/GetBlockedDomains}
     */
    GetBlockedDomains: GetBlockedDomains,
    /**
     * The GetCampaignOverview model constructor.
     * @property {module:model/GetCampaignOverview}
     */
    GetCampaignOverview: GetCampaignOverview,
    /**
     * The GetCampaignRecipients model constructor.
     * @property {module:model/GetCampaignRecipients}
     */
    GetCampaignRecipients: GetCampaignRecipients,
    /**
     * The GetCampaignStats model constructor.
     * @property {module:model/GetCampaignStats}
     */
    GetCampaignStats: GetCampaignStats,
    /**
     * The GetChildAccountCreationStatus model constructor.
     * @property {module:model/GetChildAccountCreationStatus}
     */
    GetChildAccountCreationStatus: GetChildAccountCreationStatus,
    /**
     * The GetChildDomain model constructor.
     * @property {module:model/GetChildDomain}
     */
    GetChildDomain: GetChildDomain,
    /**
     * The GetChildDomains model constructor.
     * @property {module:model/GetChildDomains}
     */
    GetChildDomains: GetChildDomains,
    /**
     * The GetChildInfo model constructor.
     * @property {module:model/GetChildInfo}
     */
    GetChildInfo: GetChildInfo,
    /**
     * The GetChildInfoApiKeys model constructor.
     * @property {module:model/GetChildInfoApiKeys}
     */
    GetChildInfoApiKeys: GetChildInfoApiKeys,
    /**
     * The GetChildInfoApiKeysV2 model constructor.
     * @property {module:model/GetChildInfoApiKeysV2}
     */
    GetChildInfoApiKeysV2: GetChildInfoApiKeysV2,
    /**
     * The GetChildInfoApiKeysV3 model constructor.
     * @property {module:model/GetChildInfoApiKeysV3}
     */
    GetChildInfoApiKeysV3: GetChildInfoApiKeysV3,
    /**
     * The GetChildInfoCredits model constructor.
     * @property {module:model/GetChildInfoCredits}
     */
    GetChildInfoCredits: GetChildInfoCredits,
    /**
     * The GetChildInfoStatistics model constructor.
     * @property {module:model/GetChildInfoStatistics}
     */
    GetChildInfoStatistics: GetChildInfoStatistics,
    /**
     * The GetChildrenList model constructor.
     * @property {module:model/GetChildrenList}
     */
    GetChildrenList: GetChildrenList,
    /**
     * The GetClient model constructor.
     * @property {module:model/GetClient}
     */
    GetClient: GetClient,
    /**
     * The GetContactCampaignStats model constructor.
     * @property {module:model/GetContactCampaignStats}
     */
    GetContactCampaignStats: GetContactCampaignStats,
    /**
     * The GetContactCampaignStatsClicked model constructor.
     * @property {module:model/GetContactCampaignStatsClicked}
     */
    GetContactCampaignStatsClicked: GetContactCampaignStatsClicked,
    /**
     * The GetContactCampaignStatsOpened model constructor.
     * @property {module:model/GetContactCampaignStatsOpened}
     */
    GetContactCampaignStatsOpened: GetContactCampaignStatsOpened,
    /**
     * The GetContactCampaignStatsTransacAttributes model constructor.
     * @property {module:model/GetContactCampaignStatsTransacAttributes}
     */
    GetContactCampaignStatsTransacAttributes: GetContactCampaignStatsTransacAttributes,
    /**
     * The GetContactCampaignStatsUnsubscriptions model constructor.
     * @property {module:model/GetContactCampaignStatsUnsubscriptions}
     */
    GetContactCampaignStatsUnsubscriptions: GetContactCampaignStatsUnsubscriptions,
    /**
     * The GetContactDetails model constructor.
     * @property {module:model/GetContactDetails}
     */
    GetContactDetails: GetContactDetails,
    /**
     * The GetContacts model constructor.
     * @property {module:model/GetContacts}
     */
    GetContacts: GetContacts,
    /**
     * The GetDeviceBrowserStats model constructor.
     * @property {module:model/GetDeviceBrowserStats}
     */
    GetDeviceBrowserStats: GetDeviceBrowserStats,
    /**
     * The GetEmailCampaign model constructor.
     * @property {module:model/GetEmailCampaign}
     */
    GetEmailCampaign: GetEmailCampaign,
    /**
     * The GetEmailCampaigns model constructor.
     * @property {module:model/GetEmailCampaigns}
     */
    GetEmailCampaigns: GetEmailCampaigns,
    /**
     * The GetEmailEventReport model constructor.
     * @property {module:model/GetEmailEventReport}
     */
    GetEmailEventReport: GetEmailEventReport,
    /**
     * The GetEmailEventReportEvents model constructor.
     * @property {module:model/GetEmailEventReportEvents}
     */
    GetEmailEventReportEvents: GetEmailEventReportEvents,
    /**
     * The GetExtendedCampaignOverview model constructor.
     * @property {module:model/GetExtendedCampaignOverview}
     */
    GetExtendedCampaignOverview: GetExtendedCampaignOverview,
    /**
     * The GetExtendedCampaignOverviewSender model constructor.
     * @property {module:model/GetExtendedCampaignOverviewSender}
     */
    GetExtendedCampaignOverviewSender: GetExtendedCampaignOverviewSender,
    /**
     * The GetExtendedCampaignStats model constructor.
     * @property {module:model/GetExtendedCampaignStats}
     */
    GetExtendedCampaignStats: GetExtendedCampaignStats,
    /**
     * The GetExtendedCampaignStatsGlobalStats model constructor.
     * @property {module:model/GetExtendedCampaignStatsGlobalStats}
     */
    GetExtendedCampaignStatsGlobalStats: GetExtendedCampaignStatsGlobalStats,
    /**
     * The GetExtendedClient model constructor.
     * @property {module:model/GetExtendedClient}
     */
    GetExtendedClient: GetExtendedClient,
    /**
     * The GetExtendedClientAddress model constructor.
     * @property {module:model/GetExtendedClientAddress}
     */
    GetExtendedClientAddress: GetExtendedClientAddress,
    /**
     * The GetExtendedContactDetails model constructor.
     * @property {module:model/GetExtendedContactDetails}
     */
    GetExtendedContactDetails: GetExtendedContactDetails,
    /**
     * The GetExtendedContactDetailsStatistics model constructor.
     * @property {module:model/GetExtendedContactDetailsStatistics}
     */
    GetExtendedContactDetailsStatistics: GetExtendedContactDetailsStatistics,
    /**
     * The GetExtendedContactDetailsStatisticsClicked model constructor.
     * @property {module:model/GetExtendedContactDetailsStatisticsClicked}
     */
    GetExtendedContactDetailsStatisticsClicked: GetExtendedContactDetailsStatisticsClicked,
    /**
     * The GetExtendedContactDetailsStatisticsDelivered model constructor.
     * @property {module:model/GetExtendedContactDetailsStatisticsDelivered}
     */
    GetExtendedContactDetailsStatisticsDelivered: GetExtendedContactDetailsStatisticsDelivered,
    /**
     * The GetExtendedContactDetailsStatisticsLinks model constructor.
     * @property {module:model/GetExtendedContactDetailsStatisticsLinks}
     */
    GetExtendedContactDetailsStatisticsLinks: GetExtendedContactDetailsStatisticsLinks,
    /**
     * The GetExtendedContactDetailsStatisticsMessagesSent model constructor.
     * @property {module:model/GetExtendedContactDetailsStatisticsMessagesSent}
     */
    GetExtendedContactDetailsStatisticsMessagesSent: GetExtendedContactDetailsStatisticsMessagesSent,
    /**
     * The GetExtendedContactDetailsStatisticsOpened model constructor.
     * @property {module:model/GetExtendedContactDetailsStatisticsOpened}
     */
    GetExtendedContactDetailsStatisticsOpened: GetExtendedContactDetailsStatisticsOpened,
    /**
     * The GetExtendedContactDetailsStatisticsUnsubscriptions model constructor.
     * @property {module:model/GetExtendedContactDetailsStatisticsUnsubscriptions}
     */
    GetExtendedContactDetailsStatisticsUnsubscriptions: GetExtendedContactDetailsStatisticsUnsubscriptions,
    /**
     * The GetExtendedContactDetailsStatisticsUnsubscriptionsAdminUnsubscription model constructor.
     * @property {module:model/GetExtendedContactDetailsStatisticsUnsubscriptionsAdminUnsubscription}
     */
    GetExtendedContactDetailsStatisticsUnsubscriptionsAdminUnsubscription: GetExtendedContactDetailsStatisticsUnsubscriptionsAdminUnsubscription,
    /**
     * The GetExtendedContactDetailsStatisticsUnsubscriptionsUserUnsubscription model constructor.
     * @property {module:model/GetExtendedContactDetailsStatisticsUnsubscriptionsUserUnsubscription}
     */
    GetExtendedContactDetailsStatisticsUnsubscriptionsUserUnsubscription: GetExtendedContactDetailsStatisticsUnsubscriptionsUserUnsubscription,
    /**
     * The GetExtendedList model constructor.
     * @property {module:model/GetExtendedList}
     */
    GetExtendedList: GetExtendedList,
    /**
     * The GetExtendedListCampaignStats model constructor.
     * @property {module:model/GetExtendedListCampaignStats}
     */
    GetExtendedListCampaignStats: GetExtendedListCampaignStats,
    /**
     * The GetFolder model constructor.
     * @property {module:model/GetFolder}
     */
    GetFolder: GetFolder,
    /**
     * The GetFolderLists model constructor.
     * @property {module:model/GetFolderLists}
     */
    GetFolderLists: GetFolderLists,
    /**
     * The GetFolders model constructor.
     * @property {module:model/GetFolders}
     */
    GetFolders: GetFolders,
    /**
     * The GetInboundEmailEvents model constructor.
     * @property {module:model/GetInboundEmailEvents}
     */
    GetInboundEmailEvents: GetInboundEmailEvents,
    /**
     * The GetInboundEmailEventsByUuid model constructor.
     * @property {module:model/GetInboundEmailEventsByUuid}
     */
    GetInboundEmailEventsByUuid: GetInboundEmailEventsByUuid,
    /**
     * The GetInboundEmailEventsByUuidAttachments model constructor.
     * @property {module:model/GetInboundEmailEventsByUuidAttachments}
     */
    GetInboundEmailEventsByUuidAttachments: GetInboundEmailEventsByUuidAttachments,
    /**
     * The GetInboundEmailEventsByUuidLogs model constructor.
     * @property {module:model/GetInboundEmailEventsByUuidLogs}
     */
    GetInboundEmailEventsByUuidLogs: GetInboundEmailEventsByUuidLogs,
    /**
     * The GetInboundEmailEventsEvents model constructor.
     * @property {module:model/GetInboundEmailEventsEvents}
     */
    GetInboundEmailEventsEvents: GetInboundEmailEventsEvents,
    /**
     * The GetIp model constructor.
     * @property {module:model/GetIp}
     */
    GetIp: GetIp,
    /**
     * The GetIpFromSender model constructor.
     * @property {module:model/GetIpFromSender}
     */
    GetIpFromSender: GetIpFromSender,
    /**
     * The GetIps model constructor.
     * @property {module:model/GetIps}
     */
    GetIps: GetIps,
    /**
     * The GetIpsFromSender model constructor.
     * @property {module:model/GetIpsFromSender}
     */
    GetIpsFromSender: GetIpsFromSender,
    /**
     * The GetList model constructor.
     * @property {module:model/GetList}
     */
    GetList: GetList,
    /**
     * The GetLists model constructor.
     * @property {module:model/GetLists}
     */
    GetLists: GetLists,
    /**
     * The GetProcess model constructor.
     * @property {module:model/GetProcess}
     */
    GetProcess: GetProcess,
    /**
     * The GetProcesses model constructor.
     * @property {module:model/GetProcesses}
     */
    GetProcesses: GetProcesses,
    /**
     * The GetReports model constructor.
     * @property {module:model/GetReports}
     */
    GetReports: GetReports,
    /**
     * The GetReportsReports model constructor.
     * @property {module:model/GetReportsReports}
     */
    GetReportsReports: GetReportsReports,
    /**
     * The GetScheduledEmailByBatchId model constructor.
     * @property {module:model/GetScheduledEmailByBatchId}
     */
    GetScheduledEmailByBatchId: GetScheduledEmailByBatchId,
    /**
     * The GetScheduledEmailByBatchIdBatches model constructor.
     * @property {module:model/GetScheduledEmailByBatchIdBatches}
     */
    GetScheduledEmailByBatchIdBatches: GetScheduledEmailByBatchIdBatches,
    /**
     * The GetScheduledEmailByMessageId model constructor.
     * @property {module:model/GetScheduledEmailByMessageId}
     */
    GetScheduledEmailByMessageId: GetScheduledEmailByMessageId,
    /**
     * The GetSendersList model constructor.
     * @property {module:model/GetSendersList}
     */
    GetSendersList: GetSendersList,
    /**
     * The GetSendersListIps model constructor.
     * @property {module:model/GetSendersListIps}
     */
    GetSendersListIps: GetSendersListIps,
    /**
     * The GetSendersListSenders model constructor.
     * @property {module:model/GetSendersListSenders}
     */
    GetSendersListSenders: GetSendersListSenders,
    /**
     * The GetSharedTemplateUrl model constructor.
     * @property {module:model/GetSharedTemplateUrl}
     */
    GetSharedTemplateUrl: GetSharedTemplateUrl,
    /**
     * The GetSmsCampaign model constructor.
     * @property {module:model/GetSmsCampaign}
     */
    GetSmsCampaign: GetSmsCampaign,
    /**
     * The GetSmsCampaignOverview model constructor.
     * @property {module:model/GetSmsCampaignOverview}
     */
    GetSmsCampaignOverview: GetSmsCampaignOverview,
    /**
     * The GetSmsCampaignRecipients model constructor.
     * @property {module:model/GetSmsCampaignRecipients}
     */
    GetSmsCampaignRecipients: GetSmsCampaignRecipients,
    /**
     * The GetSmsCampaignStats model constructor.
     * @property {module:model/GetSmsCampaignStats}
     */
    GetSmsCampaignStats: GetSmsCampaignStats,
    /**
     * The GetSmsCampaigns model constructor.
     * @property {module:model/GetSmsCampaigns}
     */
    GetSmsCampaigns: GetSmsCampaigns,
    /**
     * The GetSmsEventReport model constructor.
     * @property {module:model/GetSmsEventReport}
     */
    GetSmsEventReport: GetSmsEventReport,
    /**
     * The GetSmsEventReportEvents model constructor.
     * @property {module:model/GetSmsEventReportEvents}
     */
    GetSmsEventReportEvents: GetSmsEventReportEvents,
    /**
     * The GetSmtpTemplateOverview model constructor.
     * @property {module:model/GetSmtpTemplateOverview}
     */
    GetSmtpTemplateOverview: GetSmtpTemplateOverview,
    /**
     * The GetSmtpTemplateOverviewSender model constructor.
     * @property {module:model/GetSmtpTemplateOverviewSender}
     */
    GetSmtpTemplateOverviewSender: GetSmtpTemplateOverviewSender,
    /**
     * The GetSmtpTemplates model constructor.
     * @property {module:model/GetSmtpTemplates}
     */
    GetSmtpTemplates: GetSmtpTemplates,
    /**
     * The GetSsoToken model constructor.
     * @property {module:model/GetSsoToken}
     */
    GetSsoToken: GetSsoToken,
    /**
     * The GetStatsByBrowser model constructor.
     * @property {module:model/GetStatsByBrowser}
     */
    GetStatsByBrowser: GetStatsByBrowser,
    /**
     * The GetStatsByDevice model constructor.
     * @property {module:model/GetStatsByDevice}
     */
    GetStatsByDevice: GetStatsByDevice,
    /**
     * The GetStatsByDomain model constructor.
     * @property {module:model/GetStatsByDomain}
     */
    GetStatsByDomain: GetStatsByDomain,
    /**
     * The GetTransacAggregatedSmsReport model constructor.
     * @property {module:model/GetTransacAggregatedSmsReport}
     */
    GetTransacAggregatedSmsReport: GetTransacAggregatedSmsReport,
    /**
     * The GetTransacBlockedContacts model constructor.
     * @property {module:model/GetTransacBlockedContacts}
     */
    GetTransacBlockedContacts: GetTransacBlockedContacts,
    /**
     * The GetTransacBlockedContactsContacts model constructor.
     * @property {module:model/GetTransacBlockedContactsContacts}
     */
    GetTransacBlockedContactsContacts: GetTransacBlockedContactsContacts,
    /**
     * The GetTransacBlockedContactsReason model constructor.
     * @property {module:model/GetTransacBlockedContactsReason}
     */
    GetTransacBlockedContactsReason: GetTransacBlockedContactsReason,
    /**
     * The GetTransacEmailContent model constructor.
     * @property {module:model/GetTransacEmailContent}
     */
    GetTransacEmailContent: GetTransacEmailContent,
    /**
     * The GetTransacEmailContentEvents model constructor.
     * @property {module:model/GetTransacEmailContentEvents}
     */
    GetTransacEmailContentEvents: GetTransacEmailContentEvents,
    /**
     * The GetTransacEmailsList model constructor.
     * @property {module:model/GetTransacEmailsList}
     */
    GetTransacEmailsList: GetTransacEmailsList,
    /**
     * The GetTransacEmailsListTransactionalEmails model constructor.
     * @property {module:model/GetTransacEmailsListTransactionalEmails}
     */
    GetTransacEmailsListTransactionalEmails: GetTransacEmailsListTransactionalEmails,
    /**
     * The GetTransacSmsReport model constructor.
     * @property {module:model/GetTransacSmsReport}
     */
    GetTransacSmsReport: GetTransacSmsReport,
    /**
     * The GetTransacSmsReportReports model constructor.
     * @property {module:model/GetTransacSmsReportReports}
     */
    GetTransacSmsReportReports: GetTransacSmsReportReports,
    /**
     * The GetWebhook model constructor.
     * @property {module:model/GetWebhook}
     */
    GetWebhook: GetWebhook,
    /**
     * The GetWebhooks model constructor.
     * @property {module:model/GetWebhooks}
     */
    GetWebhooks: GetWebhooks,
    /**
     * The InlineResponse200 model constructor.
     * @property {module:model/InlineResponse200}
     */
    InlineResponse200: InlineResponse200,
    /**
     * The InlineResponse201 model constructor.
     * @property {module:model/InlineResponse201}
     */
    InlineResponse201: InlineResponse201,
    /**
     * The InlineResponse2011 model constructor.
     * @property {module:model/InlineResponse2011}
     */
    InlineResponse2011: InlineResponse2011,
    /**
     * The ManageIp model constructor.
     * @property {module:model/ManageIp}
     */
    ManageIp: ManageIp,
    /**
     * The MasterDetailsResponse model constructor.
     * @property {module:model/MasterDetailsResponse}
     */
    MasterDetailsResponse: MasterDetailsResponse,
    /**
     * The MasterDetailsResponseBillingInfo model constructor.
     * @property {module:model/MasterDetailsResponseBillingInfo}
     */
    MasterDetailsResponseBillingInfo: MasterDetailsResponseBillingInfo,
    /**
     * The MasterDetailsResponseBillingInfoAddress model constructor.
     * @property {module:model/MasterDetailsResponseBillingInfoAddress}
     */
    MasterDetailsResponseBillingInfoAddress: MasterDetailsResponseBillingInfoAddress,
    /**
     * The MasterDetailsResponseBillingInfoName model constructor.
     * @property {module:model/MasterDetailsResponseBillingInfoName}
     */
    MasterDetailsResponseBillingInfoName: MasterDetailsResponseBillingInfoName,
    /**
     * The MasterDetailsResponsePlanInfo model constructor.
     * @property {module:model/MasterDetailsResponsePlanInfo}
     */
    MasterDetailsResponsePlanInfo: MasterDetailsResponsePlanInfo,
    /**
     * The MasterDetailsResponsePlanInfoFeatures model constructor.
     * @property {module:model/MasterDetailsResponsePlanInfoFeatures}
     */
    MasterDetailsResponsePlanInfoFeatures: MasterDetailsResponsePlanInfoFeatures,
    /**
     * The Note model constructor.
     * @property {module:model/Note}
     */
    Note: Note,
    /**
     * The NoteData model constructor.
     * @property {module:model/NoteData}
     */
    NoteData: NoteData,
    /**
     * The NoteId model constructor.
     * @property {module:model/NoteId}
     */
    NoteId: NoteId,
    /**
     * The NoteList model constructor.
     * @property {module:model/NoteList}
     */
    NoteList: NoteList,
    /**
     * The Pipeline model constructor.
     * @property {module:model/Pipeline}
     */
    Pipeline: Pipeline,
    /**
     * The PipelineStage model constructor.
     * @property {module:model/PipelineStage}
     */
    PipelineStage: PipelineStage,
    /**
     * The PostContactInfo model constructor.
     * @property {module:model/PostContactInfo}
     */
    PostContactInfo: PostContactInfo,
    /**
     * The PostContactInfoContacts model constructor.
     * @property {module:model/PostContactInfoContacts}
     */
    PostContactInfoContacts: PostContactInfoContacts,
    /**
     * The PostSendFailed model constructor.
     * @property {module:model/PostSendFailed}
     */
    PostSendFailed: PostSendFailed,
    /**
     * The PostSendSmsTestFailed model constructor.
     * @property {module:model/PostSendSmsTestFailed}
     */
    PostSendSmsTestFailed: PostSendSmsTestFailed,
    /**
     * The RemainingCreditModel model constructor.
     * @property {module:model/RemainingCreditModel}
     */
    RemainingCreditModel: RemainingCreditModel,
    /**
     * The RemainingCreditModelChild model constructor.
     * @property {module:model/RemainingCreditModelChild}
     */
    RemainingCreditModelChild: RemainingCreditModelChild,
    /**
     * The RemainingCreditModelReseller model constructor.
     * @property {module:model/RemainingCreditModelReseller}
     */
    RemainingCreditModelReseller: RemainingCreditModelReseller,
    /**
     * The RemoveContactFromList model constructor.
     * @property {module:model/RemoveContactFromList}
     */
    RemoveContactFromList: RemoveContactFromList,
    /**
     * The RemoveCredits model constructor.
     * @property {module:model/RemoveCredits}
     */
    RemoveCredits: RemoveCredits,
    /**
     * The RequestContactExport model constructor.
     * @property {module:model/RequestContactExport}
     */
    RequestContactExport: RequestContactExport,
    /**
     * The RequestContactExportCustomContactFilter model constructor.
     * @property {module:model/RequestContactExportCustomContactFilter}
     */
    RequestContactExportCustomContactFilter: RequestContactExportCustomContactFilter,
    /**
     * The RequestContactImport model constructor.
     * @property {module:model/RequestContactImport}
     */
    RequestContactImport: RequestContactImport,
    /**
     * The RequestContactImportNewList model constructor.
     * @property {module:model/RequestContactImportNewList}
     */
    RequestContactImportNewList: RequestContactImportNewList,
    /**
     * The RequestSmsRecipientExport model constructor.
     * @property {module:model/RequestSmsRecipientExport}
     */
    RequestSmsRecipientExport: RequestSmsRecipientExport,
    /**
     * The ScheduleSmtpEmail model constructor.
     * @property {module:model/ScheduleSmtpEmail}
     */
    ScheduleSmtpEmail: ScheduleSmtpEmail,
    /**
     * The SendReport model constructor.
     * @property {module:model/SendReport}
     */
    SendReport: SendReport,
    /**
     * The SendReportEmail model constructor.
     * @property {module:model/SendReportEmail}
     */
    SendReportEmail: SendReportEmail,
    /**
     * The SendSms model constructor.
     * @property {module:model/SendSms}
     */
    SendSms: SendSms,
    /**
     * The SendSmtpEmail model constructor.
     * @property {module:model/SendSmtpEmail}
     */
    SendSmtpEmail: SendSmtpEmail,
    /**
     * The SendSmtpEmailAttachment model constructor.
     * @property {module:model/SendSmtpEmailAttachment}
     */
    SendSmtpEmailAttachment: SendSmtpEmailAttachment,
    /**
     * The SendSmtpEmailBcc model constructor.
     * @property {module:model/SendSmtpEmailBcc}
     */
    SendSmtpEmailBcc: SendSmtpEmailBcc,
    /**
     * The SendSmtpEmailCc model constructor.
     * @property {module:model/SendSmtpEmailCc}
     */
    SendSmtpEmailCc: SendSmtpEmailCc,
    /**
     * The SendSmtpEmailMessageVersions model constructor.
     * @property {module:model/SendSmtpEmailMessageVersions}
     */
    SendSmtpEmailMessageVersions: SendSmtpEmailMessageVersions,
    /**
     * The SendSmtpEmailReplyTo model constructor.
     * @property {module:model/SendSmtpEmailReplyTo}
     */
    SendSmtpEmailReplyTo: SendSmtpEmailReplyTo,
    /**
     * The SendSmtpEmailReplyTo1 model constructor.
     * @property {module:model/SendSmtpEmailReplyTo1}
     */
    SendSmtpEmailReplyTo1: SendSmtpEmailReplyTo1,
    /**
     * The SendSmtpEmailSender model constructor.
     * @property {module:model/SendSmtpEmailSender}
     */
    SendSmtpEmailSender: SendSmtpEmailSender,
    /**
     * The SendSmtpEmailTo model constructor.
     * @property {module:model/SendSmtpEmailTo}
     */
    SendSmtpEmailTo: SendSmtpEmailTo,
    /**
     * The SendSmtpEmailTo1 model constructor.
     * @property {module:model/SendSmtpEmailTo1}
     */
    SendSmtpEmailTo1: SendSmtpEmailTo1,
    /**
     * The SendTestEmail model constructor.
     * @property {module:model/SendTestEmail}
     */
    SendTestEmail: SendTestEmail,
    /**
     * The SendTestSms model constructor.
     * @property {module:model/SendTestSms}
     */
    SendTestSms: SendTestSms,
    /**
     * The SendTransacSms model constructor.
     * @property {module:model/SendTransacSms}
     */
    SendTransacSms: SendTransacSms,
    /**
     * The SsoTokenRequest model constructor.
     * @property {module:model/SsoTokenRequest}
     */
    SsoTokenRequest: SsoTokenRequest,
    /**
     * The SubAccountDetailsResponse model constructor.
     * @property {module:model/SubAccountDetailsResponse}
     */
    SubAccountDetailsResponse: SubAccountDetailsResponse,
    /**
     * The SubAccountDetailsResponsePlanInfo model constructor.
     * @property {module:model/SubAccountDetailsResponsePlanInfo}
     */
    SubAccountDetailsResponsePlanInfo: SubAccountDetailsResponsePlanInfo,
    /**
     * The SubAccountDetailsResponsePlanInfoCredits model constructor.
     * @property {module:model/SubAccountDetailsResponsePlanInfoCredits}
     */
    SubAccountDetailsResponsePlanInfoCredits: SubAccountDetailsResponsePlanInfoCredits,
    /**
     * The SubAccountDetailsResponsePlanInfoCreditsEmails model constructor.
     * @property {module:model/SubAccountDetailsResponsePlanInfoCreditsEmails}
     */
    SubAccountDetailsResponsePlanInfoCreditsEmails: SubAccountDetailsResponsePlanInfoCreditsEmails,
    /**
     * The SubAccountDetailsResponsePlanInfoFeatures model constructor.
     * @property {module:model/SubAccountDetailsResponsePlanInfoFeatures}
     */
    SubAccountDetailsResponsePlanInfoFeatures: SubAccountDetailsResponsePlanInfoFeatures,
    /**
     * The SubAccountDetailsResponsePlanInfoFeaturesInbox model constructor.
     * @property {module:model/SubAccountDetailsResponsePlanInfoFeaturesInbox}
     */
    SubAccountDetailsResponsePlanInfoFeaturesInbox: SubAccountDetailsResponsePlanInfoFeaturesInbox,
    /**
     * The SubAccountDetailsResponsePlanInfoFeaturesLandingPage model constructor.
     * @property {module:model/SubAccountDetailsResponsePlanInfoFeaturesLandingPage}
     */
    SubAccountDetailsResponsePlanInfoFeaturesLandingPage: SubAccountDetailsResponsePlanInfoFeaturesLandingPage,
    /**
     * The SubAccountDetailsResponsePlanInfoFeaturesUsers model constructor.
     * @property {module:model/SubAccountDetailsResponsePlanInfoFeaturesUsers}
     */
    SubAccountDetailsResponsePlanInfoFeaturesUsers: SubAccountDetailsResponsePlanInfoFeaturesUsers,
    /**
     * The SubAccountUpdatePlanRequest model constructor.
     * @property {module:model/SubAccountUpdatePlanRequest}
     */
    SubAccountUpdatePlanRequest: SubAccountUpdatePlanRequest,
    /**
     * The SubAccountUpdatePlanRequestCredits model constructor.
     * @property {module:model/SubAccountUpdatePlanRequestCredits}
     */
    SubAccountUpdatePlanRequestCredits: SubAccountUpdatePlanRequestCredits,
    /**
     * The SubAccountUpdatePlanRequestFeatures model constructor.
     * @property {module:model/SubAccountUpdatePlanRequestFeatures}
     */
    SubAccountUpdatePlanRequestFeatures: SubAccountUpdatePlanRequestFeatures,
    /**
     * The SubAccountsResponse model constructor.
     * @property {module:model/SubAccountsResponse}
     */
    SubAccountsResponse: SubAccountsResponse,
    /**
     * The SubAccountsResponseSubAccounts model constructor.
     * @property {module:model/SubAccountsResponseSubAccounts}
     */
    SubAccountsResponseSubAccounts: SubAccountsResponseSubAccounts,
    /**
     * The Task model constructor.
     * @property {module:model/Task}
     */
    Task: Task,
    /**
     * The TaskList model constructor.
     * @property {module:model/TaskList}
     */
    TaskList: TaskList,
    /**
     * The TaskReminder model constructor.
     * @property {module:model/TaskReminder}
     */
    TaskReminder: TaskReminder,
    /**
     * The TaskTypes model constructor.
     * @property {module:model/TaskTypes}
     */
    TaskTypes: TaskTypes,
    /**
     * The UpdateAttribute model constructor.
     * @property {module:model/UpdateAttribute}
     */
    UpdateAttribute: UpdateAttribute,
    /**
     * The UpdateAttributeEnumeration model constructor.
     * @property {module:model/UpdateAttributeEnumeration}
     */
    UpdateAttributeEnumeration: UpdateAttributeEnumeration,
    /**
     * The UpdateBatchContacts model constructor.
     * @property {module:model/UpdateBatchContacts}
     */
    UpdateBatchContacts: UpdateBatchContacts,
    /**
     * The UpdateBatchContactsContacts model constructor.
     * @property {module:model/UpdateBatchContactsContacts}
     */
    UpdateBatchContactsContacts: UpdateBatchContactsContacts,
    /**
     * The UpdateBatchContactsModel model constructor.
     * @property {module:model/UpdateBatchContactsModel}
     */
    UpdateBatchContactsModel: UpdateBatchContactsModel,
    /**
     * The UpdateCampaignStatus model constructor.
     * @property {module:model/UpdateCampaignStatus}
     */
    UpdateCampaignStatus: UpdateCampaignStatus,
    /**
     * The UpdateChild model constructor.
     * @property {module:model/UpdateChild}
     */
    UpdateChild: UpdateChild,
    /**
     * The UpdateChildAccountStatus model constructor.
     * @property {module:model/UpdateChildAccountStatus}
     */
    UpdateChildAccountStatus: UpdateChildAccountStatus,
    /**
     * The UpdateChildDomain model constructor.
     * @property {module:model/UpdateChildDomain}
     */
    UpdateChildDomain: UpdateChildDomain,
    /**
     * The UpdateContact model constructor.
     * @property {module:model/UpdateContact}
     */
    UpdateContact: UpdateContact,
    /**
     * The UpdateEmailCampaign model constructor.
     * @property {module:model/UpdateEmailCampaign}
     */
    UpdateEmailCampaign: UpdateEmailCampaign,
    /**
     * The UpdateEmailCampaignRecipients model constructor.
     * @property {module:model/UpdateEmailCampaignRecipients}
     */
    UpdateEmailCampaignRecipients: UpdateEmailCampaignRecipients,
    /**
     * The UpdateEmailCampaignSender model constructor.
     * @property {module:model/UpdateEmailCampaignSender}
     */
    UpdateEmailCampaignSender: UpdateEmailCampaignSender,
    /**
     * The UpdateList model constructor.
     * @property {module:model/UpdateList}
     */
    UpdateList: UpdateList,
    /**
     * The UpdateSender model constructor.
     * @property {module:model/UpdateSender}
     */
    UpdateSender: UpdateSender,
    /**
     * The UpdateSmsCampaign model constructor.
     * @property {module:model/UpdateSmsCampaign}
     */
    UpdateSmsCampaign: UpdateSmsCampaign,
    /**
     * The UpdateSmtpTemplate model constructor.
     * @property {module:model/UpdateSmtpTemplate}
     */
    UpdateSmtpTemplate: UpdateSmtpTemplate,
    /**
     * The UpdateSmtpTemplateSender model constructor.
     * @property {module:model/UpdateSmtpTemplateSender}
     */
    UpdateSmtpTemplateSender: UpdateSmtpTemplateSender,
    /**
     * The UpdateWebhook model constructor.
     * @property {module:model/UpdateWebhook}
     */
    UpdateWebhook: UpdateWebhook,
    /**
     * The UploadImageModel model constructor.
     * @property {module:model/UploadImageModel}
     */
    UploadImageModel: UploadImageModel,
    /**
     * The UploadImageToGallery model constructor.
     * @property {module:model/UploadImageToGallery}
     */
    UploadImageToGallery: UploadImageToGallery,
    /**
     * The AccountApi service constructor.
     * @property {module:api/AccountApi}
     */
    AccountApi: AccountApi,
    /**
     * The CompaniesApi service constructor.
     * @property {module:api/CompaniesApi}
     */
    CompaniesApi: CompaniesApi,
    /**
     * The ContactsApi service constructor.
     * @property {module:api/ContactsApi}
     */
    ContactsApi: ContactsApi,
    /**
     * The ConversationsApi service constructor.
     * @property {module:api/ConversationsApi}
     */
    ConversationsApi: ConversationsApi,
    /**
     * The DealsApi service constructor.
     * @property {module:api/DealsApi}
     */
    DealsApi: DealsApi,
    /**
     * The EmailCampaignsApi service constructor.
     * @property {module:api/EmailCampaignsApi}
     */
    EmailCampaignsApi: EmailCampaignsApi,
    /**
     * The FilesApi service constructor.
     * @property {module:api/FilesApi}
     */
    FilesApi: FilesApi,
    /**
     * The InboundParsingApi service constructor.
     * @property {module:api/InboundParsingApi}
     */
    InboundParsingApi: InboundParsingApi,
    /**
     * The MasterAccountApi service constructor.
     * @property {module:api/MasterAccountApi}
     */
    MasterAccountApi: MasterAccountApi,
    /**
     * The NotesApi service constructor.
     * @property {module:api/NotesApi}
     */
    NotesApi: NotesApi,
    /**
     * The ProcessApi service constructor.
     * @property {module:api/ProcessApi}
     */
    ProcessApi: ProcessApi,
    /**
     * The ResellerApi service constructor.
     * @property {module:api/ResellerApi}
     */
    ResellerApi: ResellerApi,
    /**
     * The SMSCampaignsApi service constructor.
     * @property {module:api/SMSCampaignsApi}
     */
    SMSCampaignsApi: SMSCampaignsApi,
    /**
     * The SendersApi service constructor.
     * @property {module:api/SendersApi}
     */
    SendersApi: SendersApi,
    /**
     * The TasksApi service constructor.
     * @property {module:api/TasksApi}
     */
    TasksApi: TasksApi,
    /**
     * The TransactionalEmailsApi service constructor.
     * @property {module:api/TransactionalEmailsApi}
     */
    TransactionalEmailsApi: TransactionalEmailsApi,
    /**
     * The TransactionalSMSApi service constructor.
     * @property {module:api/TransactionalSMSApi}
     */
    TransactionalSMSApi: TransactionalSMSApi,
    /**
     * The WebhooksApi service constructor.
     * @property {module:api/WebhooksApi}
     */
    WebhooksApi: WebhooksApi
  };

  return exports;
}));
